SwiftMessageBar/Message.swift

```diff
@@ -23,13 +19,25 @@ final class Message: UIView {
   private var titleFontColor: UIColor!
   private var messageFontColor: UIColor!
   private var icon: UIImage?
-  var isHit: Bool = false
+  var isHit = false
   private(set) var callback: Callback?
   private(set) var duration: TimeInterval!
-  private(set) var dismiss: Bool = true
+  private(set) var dismiss = true
   private var languageDirection: NSLocale.LanguageDirection!
   private var titleFont: UIFont!
   private var messageFont: UIFont!
+  private var accessoryView: UIView?
+
+  public lazy var contentStackView: UIStackView = {
+    let contentView = UIStackView(frame: bounds)
+    contentView.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+    contentView.isLayoutMarginsRelativeArrangement = true
+    contentView.axis = .horizontal
+    contentView.spacing = 10
+
+    usesAutoLayout(true)
```

Comment:
Which view is this call to `usesAutoLayout` referring to?

Reply:
I removed it.